engine/include/cubos/engine/plugins/env_settings.hpp

```diff
@@ -0,0 +1,15 @@
+#ifndef CUBOS_ENGINE_PLUGINS_ENV_SETTINGS_HPP
+#define CUBOS_ENGINE_PLUGINS_ENV_SETTINGS_HPP
+
+#include <cubos/engine/cubos.hpp>
+
+using namespace cubos::engine;
+
+namespace cubos::engine::plugins
+{
+    /// Plugin to handle environment settings, provided via command line.
```

Comment:
Explain that the `Arguments` resource needs to have been added, otherwise this will crash, and that it is added when argc, argv are passed to `Cubos` ctor